.github/pull_request_template.md

```diff
@@ -1,11 +1,7 @@
 ### Pull Request checklist ###
 <!-- Before submitting the PR, please address each item -->
 - [ ] **Quality**: This PR builds and tests run cleanly
-  - `cargo test --all` produces no test failures
-  - `cargo clippy --all --all-targets --all-features` runs without emitting any warnings
-  - `cargo fmt` does not produce any changes to the code
-  - `./gradlew ktlint detekt` runs without emitting any warnings
-  - `swiftformat --swiftversion 4 megazords components/*/ios && swiftlint` runs without emitting any warnings or producing changes
+  - `automation/all_tests.sh` runs to completion and produces no failures
```

Comment:
We've been talking about putting all these checks into a single script for ages, trying to get my head around it all made me finally want to do it. Also, I discovered some things that weren't in this list but *were* in other places (like the contributing docs).